ncm-ceph/src/main/perl/Ceph/commands.pm

```diff
@@ -161,8 +161,8 @@ sub ssh_known_keys {
         if (!$lines) {
```

Comment:
Sorry, github isn't showing you what I meant.  I meant this chunk (expand the context):

```perl  
my $cmd = ['/usr/bin/ssh-keygen', '-F', $host];
my $output = $self->run_command_as_ceph($cmd);
# Count the lines of the output
my $lines = $output =~ tr/\n//;
if (!$lines) {
```

Why do you need to count lines at all?  It looks to me like you're doing something if there is output and something if there isn't.